config/initializers/devise_monkeypatch.rb

```diff
@@ -0,0 +1,15 @@
+module Devise
```

Comment:
Should this have a reason/explanation for this monkey patch? I'm not sure what's going on here...